ui_framework/doc_site/src/views/gallery/gallery.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import {
+  KuiGallery,
+  KuiGalleryButton,
+  KuiGalleryButtonIcon,
+  KuiGalleryButtonImage,
+  KuiGalleryButtonLabel
+} from '../../../../components';
+
+export default () => {
+  /**
+  * These styles are just for demonstration purposes. It is recommended to use
+  * properly named classes instead of CSS properties in production code.
+  */
+  const imageStyle = {
+    backgroundColor: 'lightgray'
+  };
+
+  return (
+    <div>
+      <div className="kuiVerticalRhythm">
+        <div className="kuiSubTitle">
+          Some items
+        </div>
```

Comment:
Can we change this to an `<h2>`? I think I made a mistake when I made this a div originally.